docs/development/deployment.md

```diff
@@ -60,6 +61,204 @@ If you want to reset your environment you can run the `hack/util-update-app-of-a
 
 Note running these scripts in a clone repo will have no effect as the repo will remain `https://github.com/redhat-appstudio/infra-deployments.git`
 
+### `preview.env` instructions
+
+I want a development deployment of StoneSoup where I can:
+
+* Run simple builds
+* Onboard a repository to Pipelines as Code
+
+What environment variables do I need to set in `preview.env` before I run the preview script?
+How to set up all the prerequisites?
+
+| Variable                      | How to set up                     |
+| ----------------------------- | --------------------------------- |
+| `MY_GIT_FORK_REMOTE`          | [Fork the repo](#fork-the-repo)   |
+| `MY_GITHUB_ORG`               | [GitHub setup](#github-setup)     |
+| `MY_GITHUB_TOKEN`             | [GitHub setup](#github-setup)     |
+| `IMAGE_CONTROLLER_QUAY_ORG`   | [Quay setup](#quay-setup)         |
+| `IMAGE_CONTROLLER_QUAY_TOKEN` | [Quay setup](#quay-setup)         |
+| `PAC_GITHUB_APP_PRIVATE_KEY`  | [PaC setup](#pac-setup)           |
+| `PAC_GITHUB_APP_ID`           | [PaC setup](#pac-setup)           |
+
+#### Fork the repo
+
+If you haven't done so yet, fork <https://github.com/redhat-appstudio/infra-deployments> and clone
+it locally. Copy [`hack/preview-template.env`][preview-template.env] to `hack/preview.env` - that's
+where you want to set all the environment variables.
+
+The first one is `MY_GIT_FORK_REMOTE` - the name of the git remote which points to your fork.
+
+```shell
+$ git remote -v
+origin  git@github.com:<gh_username>/infra-deployments.git (fetch)
+origin  git@github.com:<gh_username>/infra-deployments.git (push)
+upstream    https://github.com/redhat-appstudio/infra-deployments.git (fetch)
+upstream    https://github.com/redhat-appstudio/infra-deployments.git (push)
+```
+
+```shell
+export MY_GIT_FORK_REMOTE=origin
```

Comment:
How does this work?

Reply:
Ah, I see..it's the actual line that's in the preview.env file. Confused me a bit :D So maybe I would make it even more explicit here?

Reply:
Good point, I'll try to make it more obvious